src/Features/PaymentGatewaySuggestions/DefaultPaymentGateways.php

```diff
@@ -95,14 +96,16 @@ public static function get_all() {
 				),
 			),
 			array(
-				'id'         => 'woo-mercado-pago-custom',
-				'title'      => __( 'Mercado Pago Checkout Pro & Custom', 'woocommerce-admin' ),
-				'content'    => __( 'Accept credit and debit cards, offline (cash or bank transfer) and logged-in payments with money in Mercado Pago. Safe and secure payments with the leading payment processor in LATAM.', 'woocommerce-admin' ),
-				'image'      => plugins_url( 'images/onboarding/mercadopago.png', WC_ADMIN_PLUGIN_FILE ),
-				'plugins'    => array( 'woocommerce-mercadopago' ),
-				'is_visible' => array(
+				'id'               => 'woo-mercado-pago-custom',
+				'title'            => __( 'Mercado Pago Checkout Pro & Custom', 'woocommerce-admin' ),
+				'content'          => __( 'Accept credit and debit cards, offline (cash or bank transfer) and logged-in payments with money in Mercado Pago. Safe and secure payments with the leading payment processor in LATAM.', 'woocommerce-admin' ),
+				'image'            => plugins_url( 'images/onboarding/mercadopago.png', WC_ADMIN_PLUGIN_FILE ),
+				'plugins'          => array( 'woocommerce-mercadopago' ),
+				'is_visible'       => array(
 					self::get_rules_for_countries( array( 'AR', 'BR', 'CL', 'CO', 'MX', 'PE', 'UY' ) ),
 				),
+				'is_recommended'   => 2,
```

Comment:
Nice!  My only comment is that `is_recommended` looks weird here and I would expect a boolean value.  `recommendation_priority` or `recommendation_level` might be more semantic, but those are also a mouthful.  Open to any other ideas here.

Reply:
Good call @joshuatf , just changed to `recommendation_priority` since that seemed the most semantic.